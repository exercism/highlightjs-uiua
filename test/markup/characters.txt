@a @Z @  @\n @\s @\xae @\x1F @\u2665 @\u{1f600}
