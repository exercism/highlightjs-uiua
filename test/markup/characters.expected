<span class="hljs-string">@a</span> <span class="hljs-string">@Z</span> <span class="hljs-string">@ </span> <span class="hljs-string">@\n</span> <span class="hljs-string">@\s</span> <span class="hljs-string">@\xae</span> <span class="hljs-string">@\x1F</span> <span class="hljs-string">@\u2665</span> <span class="hljs-string">@\u{1f600}</span>
